repeating symbols.py

```diff
@@ -0,0 +1,14 @@
+user = str(input("Введите предложение\n"))
+# очистим от пробелов здесь, чтобы они не попали в список уникальных
+# можно и в самом конце очистить от них
+user = user.replace(" ", "")
+# создаём пустой список. Здесь будем хранить уникальные элементы
+noRepeat = []
+# по очереди проверяем есть ли символ в списке уникальных

```

Comment:
redundant comments - we shouldn't comment every line

Reply:
fixed in this particular file.. Would comment less in future.